### PR TITLE
Prepare to update bundle build by storing CMA version of CSV in this repo (part 2)

### PR DESCRIPTION
**Pull request by joelsmith**
The commit from this PR should be merged with a commit of the same name on rebase. These files were mistakenly dropped during a rebase.

### CODE REVIEW

File: cmd/testutil/json2yaml/json2yaml.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package main
```
Comment (jrvaldes) [inline cmd/testutil/json2yaml/json2yaml.go:1]: I recommend adding a ref in the commit message. i.e. https://github.com/openshift/custom-metrics-autoscaler-operator/commit/d13a2a74b423c67408c929f9a45855460a508df5

Comment (jrvaldes) [inline cmd/testutil/json2yaml/json2yaml.go:1]: Update the PR title to match the first PR.

**Review by jrvaldes — COMMENTED** [review]
LGTM

File: cmd/testutil/json2yaml/json2yaml.go (hunk 1 above)
Comment (joelsmith) [reply cmd/testutil/json2yaml/json2yaml.go:1]: Done, thanks!

Comment (joelsmith) [reply cmd/testutil/json2yaml/json2yaml.go:1]: Done, thanks!

**Pull request merged**